front/src/modules/ui/input/url/components/URLTextInputDisplay.tsx

```diff
@@ -1,9 +1,9 @@
 import { MouseEvent } from 'react';
 import styled from '@emotion/styled';
 
-import { RawLink } from '@/ui/link/components/RawLink';
+import { RoundedLink } from '@/ui/link/components/RoundedLink';
 
-const StyledRawLink = styled(RawLink)`
+const StyledRawLink = styled(RoundedLink)`
   overflow: hidden;
 
   a {
```

Comment:
this a styling is not needed! It should already be handled by RoundedLink itself

Reply:
overflow property too